src/components/PageSections/SecondaryContentBanner/index.js

```diff
@@ -1,9 +0,0 @@
-import React from 'react';
-
-const SecondaryContentBanner = ({ children }) => (
```

Comment:
What I remember was we were intending to change it b/c you thought the we thought the name wasn't descriptive but since we got in a crunch for user testing it stayed the same. I looked back on my photos of our dev/design review where we named components just now, and we labelled it pageNotification. I went with PageSubBanner because I forgot I had those photos and it appeared below the main banner. But I can poke at Celine and see what she thinks.

Reply:
Made a new PR with this renamed to PageNotificationBanner. 
https://github.com/cityofaustin/janis/pull/237